_develop/connect/use-with-clarity.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: learn
+permalink: /:collection/:path.html
+---
+# Usage with Clarity
+{:.no_toc}
+
+With Connect, you can interact with the Stacks 2.0 blockchain. You can allow your users to send transactions and interact with smart contracts.
+
+* TOC
+{:toc}
+
+## How it works
+
+For your app's users to be able to execute a smart contract function, they need to sign and broadcast a transaction. It's important that users remain control of the private keys that sign these transactions. Connect provides an easy-to-use workflow that allows your users to securely sign transactions.
+
+Connect allows you to open the authenticator with parameters indicating the details of the transaction - like the smart contract address, function name, and specific arguments. Your users get the chance to see these details, and then sign and broadcast the transaction in a single click. Their transaction will be securely signed and broadcasted onto the Stacks blockchain. After this is done, a callback is fired to allow you to update your app.
+
+## API
```

Comment:
Can we call this something else? I feel like this might reason people to think of web APIs. How about just interface? Or Usage?

Reply:
Changed to `Usage`, agreed that's better :+1: